orchestration/src/main/java/com/sap/ai/sdk/orchestration/client/model/AzureThreshold.java

```diff
@@ -14,11 +14,9 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
+import javax.annotation.Nonnull;
 
-/**
- * Threshold for the filter. Setting it to &#x60;0&#x60; blocks content with low severity, whereas
```

Comment:
why is the Javadoc gone?

Reply:
The orchestration team removed the doc from the spec